doc/user_manual_en.md

```diff
@@ -239,6 +240,14 @@ Additionally, qTox supports three modes of Markdown parsing:
 
 *Note that any change in Markdown preference will require a restart.*
 
+## Quotes
+
+qTox has feature to quote selected text in chat window:
+
+1. Select the text you want to quote.
+2. Right-click on the selected text and choose "Quote selected text" in the context menu. Also you can use `ALT` + `q` shortcut.
+3. Selected text will be automatically quoted into a message input area in a pretty formatting.
```

Comment:
"Selected text will be automatically quoted into the message input area in a pretty formatting." ?